staging/src/k8s.io/code-generator/cmd/client-gen/generators/fake/generator_fake_for_type.go

```diff
@@ -327,7 +330,7 @@ var listSubresourceTemplate = `
 func (c *Fake$.type|publicPlural$) List($.type|private$Name string, opts $.ListOptions|raw$) (result *$.resultType|raw$List, err error) {
```

Comment:
listSubresourceTemplate seems questionable, since subresource is always on a get